operations/mimir-mixin/config.libsonnet

```
@@ -188,6 +188,12 @@
       namespace_query: 'cortex_build_info{%s=~"$cluster"}' % $._config.per_cluster_label,
     },
 
+    // Used to add extra labels to all alerts, carefull can override default labels too
```

```suggestion
    // Used to add extra labels to all alerts. Careful: takes precedence over default labels.
```

can you carry out the same on the one below?

---

Done